x/concentrated-liquidity/types/pool.go

```diff
@@ -14,6 +14,7 @@ type ConcentratedPoolExtension interface {
 	GetToken1() string
 	GetCurrentSqrtPrice() sdk.Dec
 	GetCurrentTick() sdk.Int
+	GetPrecisionFactorAtPriceOne() sdk.Int
```

Comment:
If it wouldn't be too much rewiring work, I'd propose changing `PriceOne` to something more generally understandable like `StartPrice` (no strong preference on actual name, just `PriceOne` feels unusally vague)

Reply:
We need to define k at whatever the spot price of one is. I feel that StartPrice does not capture this requirement